tests/werewolves-assistant.ts

```diff
@@ -0,0 +1,17 @@
+import { runInRepo } from '../utils.ts'
+import type { RunOptions } from '../types.ts'
+
+export async function test(options: RunOptions) {
+	await runInRepo({
+		...options,
+		repo: 'antoinezanardi/werewolves-assistant-web',
+		build: ['build'],
+		test: [
+			'lint:fix',
```

Comment:
Do you think the lint command would help finding regressions for a new nuxt version?
If not, we could remove it here ☺️

Reply:
You're right, it was not relevant, I removed it 😉